config.default

```diff
@@ -122,6 +122,14 @@ overwriteMpasClimatology = False
 #   'bilinear', 'neareststod' (nearest neighbor) or 'conserve'
 mpasInterpolationMethod = bilinear
 
+# should remapping be performed with ncremap or with the Remapper class
+# directly in MPAS-Analysis
+useNcremap = False
```

Comment:
@milenaveneziani asked me to keep the option of using `ncreamp` so we can see what is faster, particularly at high resolution.  I have made the default to use the new remapper because I think the results are better, particularly with masking (e.g. MLD).